src/report/api/serializers.py

```diff
@@ -5,6 +5,8 @@
     ShelfObject
 from report.models import ObjectChangeLogReportBuilder
 from django.utils.translation import gettext_lazy as _
+import logging
```

Comment:
logging and logger are not used on this file, so we should probably delete this 2 lines of code to keep it clean.